src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/OrganizationUserUserDetailsQuery.cs

```diff
@@ -47,9 +44,9 @@ public async Task<IEnumerable<OrganizationUserUserDetailsQueryResponse>> GetOrga
                     userPermissions.DeleteAssignedCollections = false;
                 }
 
-                return orgUser;
+                return o;
             });
-        var responses = await Task.WhenAll(responseTasks);
+        var responses = responseTasks;
```

Comment:
`responseTasks` is no longer an IEnumerable of Tasks, so its name is no longer accurate, and assigning it between variables here isn't doing anything. You can just `return organizationUsers.Select...` directly.